onnx/test/shape_inference_test.py

```diff
@@ -39,8 +41,8 @@ def _make_graph(self,
                 input_value_infos.append(seed_value_info)
             else:
                 value_info.append(seed_value_info)
-                input_value_infos.append(make_tensor_value_info('SEED_' + seed_name, TensorProto.UNDEFINED, ()))
-                input_value_infos.append(make_tensor_value_info('UNKNOWN_SHAPE_' + seed_name, TensorProto.UNDEFINED, ()))
+                input_value_infos.append(make_tensor_value_info('SEED_' + seed_name, proto_type, ()))
+                input_value_infos.append(make_tensor_value_info('UNKNOWN_SHAPE_' + seed_name, proto_type, ()))
```

Comment:
shape should have a a fixed type (INT64) independent of seed-value's type: see: https://github.com/onnx/onnx/blob/master/docs/Operators.md#Reshape

Reply:
Great catch. Set the shape correctly and then some tests don't need to be changed.